### DISCUSSION TITLE
Screen cannot set temperature

### DISCUSSION THREAD

**Discussion by liqijian101** [luc-github/ESP3D-TFT#27] [Q&A] [answered]
The screen can only control axis movement.Unable to set temperature and wind speed.
Do you test temperature and wind speed。
My printer only has one nozzle, but there are two on the screen. Is it related to this

**Comment by luc-github** [comment]
you go too fast man, https://github.com/luc-github/ESP3D-TFT/discussions/1#discussioncomment-6832913
only jog is currently implemented

**Reply by luc-github** [reply, marked as answer]
this should fix part of your problem
https://github.com/luc-github/ESP3D-TFT/discussions/1#discussioncomment-6858847

**Reply by liqijian101** [reply]
thanks，It can control the temperature now。Looking forward to more improvements

**Reply by luc-github** [reply]
Speed is now implemented
